set.mm

```diff
@@ -422317,7 +422317,13 @@ or axiom of dependent choice ( ~ ax-dc ) can be used instead.
        and has no wff variables, can be proved *without* using
        ~ ax-10 through ~ ax-13 , by invoking ~ ax10w through ~ ax13w .
        We encourage proving theorems *without* ~ ax-10 through ~ ax-13
-       and moving them up to the ~ ax-4 through ~ ax-9 section.
+       and moving them up to the ~ ax-4 through ~ ax-9 section.</p>
+       <p>For these purposes, ~ df-clel , ~ df-cleq ,
+       and ~ df-clab should be considered axioms rather than definitions.
```

Comment:
df-clab is actually conservative

proof: wcel is only used by wel at the time of df-clab. df-clab is not ambiguous with wel, so wcel can be considered to have a piecewise definition with wel and df-clab (at the point of df-clab)

Reply:
OK, I wasn't sure whether to include `df-clab` just because the definition checker can't check it, or omit it because I had no knowledge of an interaction with axiom usage.

I've updated this pull request to omit it.